README.md

```diff
@@ -1 +1,10 @@
-# catfoodcalc
\ No newline at end of file
+# catfoodcalc
```

Comment:
I firmly believe that a good README with a description, build instruction, install instruction and a TODO list makes work on your project more straightforward.

It's good to start with it!